components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarFeature.kt

```diff
@@ -52,6 +52,7 @@ class ToolbarFeature(
      */
     override fun stop() {
         presenter.stop()
+        toolbar.onStop()
```

Comment:
@mauryat For testing this you could create a `ToolbarFeatureTest` file and  mock the `toolbar` to verify that every time that `ToolbarFeature#stop` is called `toolbar.onStop()` gets called too :)